### ISSUE TITLE
Minor mistakes in EPUB AT 1.1

### ISSUE THREAD

**Issue by typopaul** [mattgarrish/epub-revision#2611]
Since I do not have the experience with pull-request, I would like to give this infomation to someone who has an executive role in editing this document:

I found two minor mistakes in the EPUB Accessibility Techniques 1.1.
https://www.w3.org/TR/epub-a11y-tech-11/

Section 3.6, example 6 -- Cypher "3" typo.
> <meta3

should be

> <meta

Section 4.4.2 -- Words "to need" should be switched.
> EPUB creators also to need chunk

should be

> EPUB creators also need to chunk

**Comment by mattgarrish** [comment]
Thanks for the report! I've opened #2612 to get these fixed.
